R/check.R

```diff
@@ -193,7 +193,7 @@ coerce_dt <- function(
           "{toString(required_cols[!(required_cols %in% colnames(dt))])}",
           "but are not present among ",
           "{toString(colnames(dt))}",
-          "(all `required_cols`: {toString(required_cols)})",
+          "(all `required_cols`: {toString(required_cols)})"
```

Comment:
```suggestion
          "(all {.arg required_cols}: {toString(required_cols)})"
```
... should be using the supported inline formatting for `cli`: https://cli.r-lib.org/reference/inline-markup.html